testing/README.md

```diff
@@ -2,26 +2,27 @@
 
 ## Philosophy
 * Test code is a first-class part of the deliverable, not an optional nice-to-have.
-* When scopeing / estimating work: always factor in testing. 
-* Untestable code = bad code.
+* When scoping / estimating work: always factor in testing. 
 * Insufficent test coverage is better than no test coverage.
+* Prefer [Use Case Coverage over Code Coverage](https://kentcdodds.com/blog/how-to-know-what-to-test) (no need for 100% code coverage).
 * Additional history and guiding principles are in [this "manitesto" doc](https://bitbucket.org/blog/save-time-with-default-pull-request-descriptions)
```

Comment:
@emersonthis I did have one question for you. Is this the correct URL for the "manifesto" doc? I visited the linked page but I wasn't sure if it was the correct resource. 🤔

Reply:
https://docs.google.com/document/d/1XWx0GZLndtPF4-cwBeHii85osRj0ROPrflBF3DAVewA/edit

Reply:
Woops. It was definitely wrong and the URL Caleb added is correct. Thanks!